examples/FirebasePush_ESP8266/FirebasePush_ESP8266.ino

```diff
@@ -20,8 +20,8 @@
 #include <Firebase.h>
 
 // create firebase client.
-Firebase fbase("example.firebaseio.com")
-         .auth("secret_or_token");
+Firebase fbase = Firebase("example.firebaseio.com")
```

Comment:
This syntax is odd for c++. You are basically telling the compiled to create a empty Firebase object "fbase" then copy another anonymous Firebase object into it. So two objects + copy constructor. Copy elision should optimize this away but depending on a compiler optimization looks odd to c++ eyes.